pages/en/blog/vulnerability/july-2024-security-releases.md

```diff
@@ -1,12 +1,16 @@
 ---
 date: 2024-07-02T03:00:00.000Z
 category: vulnerability
-title: Tuesday, July 2, 2024 Security Releases
+title: Thursday, July 4, 2024 Security Releases
 slug: july-2024-security-releases
 layout: blog-post
 author: The Node.js Project
 ---
 
+## (Update 04-Jul-2024) Security Release target July 4th
```

Comment:
Should this be
```suggestion
## (Update 02-Jul-2024) Security Release target July 4th
```
? (i.e. the update was made today, announcing the new date of 4th)